lib/KeySharedPolicy.cc

```diff
@@ -51,7 +51,11 @@ KeySharedPolicy &KeySharedPolicy::setAllowOutOfOrderDelivery(bool allowOutOfOrde
 bool KeySharedPolicy::isAllowOutOfOrderDelivery() const { return impl_->allowOutOfOrderDelivery; }
 
 KeySharedPolicy &KeySharedPolicy::setStickyRanges(std::initializer_list<StickyRange> ranges) {
-    if (ranges.size() == 0) {
+    return this->setStickyRanges(v(ranges));
```

Comment:
> This should be fixed now.

Actually it's not fixed. The `v(ranges)` here is not a valid C++ semantic.

Reply:
Oops. Deleted the wrong thing. I just fixed this and recommitted.